src/Microsoft.TestPlatform.Common/Interfaces/Engine/ITestCaseEventsHandler.cs

```diff
@@ -1,16 +1,63 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Engine
+namespace Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Interfaces
 {
+    using System;
+
+    using Microsoft.VisualStudio.TestPlatform.Common.Interfaces.Engine;
     using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection;
 
     /// <summary>
-    /// The Test Case level events.
+    /// The Test Case level events handler.
     /// </summary>
     public interface ITestCaseEventsHandler
```

Comment:
>ITestCaseEventsHandler [](start = 21, length = 22)

What exactly is the responsibility of this interface? Looks muddled up and is starving for documentation.